sentry_kube/cli/apply.py

```diff
@@ -143,9 +147,13 @@ def _run_kubectl_diff(kubectl_cmd: List[str], important_diffs_only: bool) -> str
             new_env["ORIG_KUBECTL_EXTERNAL_DIFF"] = orig_kubectl_external_diff
 
         # Inject our wrapper into KUBECTL_EXTERNAL_DIFF env to filter out unwanted info
-        kubectl_external_diff_cmd = os.path.join(
-            workspace_root(), "bin/important-diffs-only"
-        )
+
+        # Find out where the important-diffs-only script is located
+        binary_name = "important-diffs-only"
+        kubectl_external_diff_cmd = shutil.which(binary_name)
+        if not kubectl_external_diff_cmd:
+            raise click.ClickException(f"Could not find {binary_name} in PATH")
```

Comment:
This is the crux of the change to dynamically find out where the binary is

Reply:
Is `important-diffs-only` on PATH?  I don't think it is so which won't find it.

Reply:
It will be in the virtual environment's path. Added the test i did in the PR description. But commenting here as well

```shell
 ~/w/ops  master <3 ?5  python
Python 3.11.6 (main, Oct  2 2023, 20:31:07) [Clang 16.0.3 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import shutil
>>>
>>> binary_name = 'important-diffs-only'
>>>
>>> # Find the path of the binary
>>> binary_path = shutil.which(binary_name)
>>>
>>> # Print the path, or a message if it's not found
>>> if binary_path:
...     print(f"The path of '{binary_name}' is: {binary_path}")
... else:
...     print(f"'{binary_name}' not found in the system's PATH.")
...
The path of 'important-diffs-only' is: /Users/nikharsaxena/workspace/ops/.venv-sentry-kube/bin/important-diffs-only
```